EIPS/eip-8743.md

```diff
@@ -0,0 +1,228 @@
+---
+eip: 8743
```

Comment:
```suggestion
eip: 7747
```
Assigning next sequential EIP/ERC/RIP number. Numbers are no longer the issue number.

Please also update the filename.